modules/dnn/src/dnn.cpp

```diff
@@ -1245,6 +1246,7 @@ struct Net::Impl
                          nextData &&
                         (!nextData->type.compare("ReLU") ||
                          !nextData->type.compare("ChannelsPReLU") ||
+                         !nextData->type.compare("TanH") ||
```

Comment:
I believe this form `0 == nextData->type.compare("...")` looks more clear than negative `compare()` call

Please update the comment above too.

Reply:
yes,  I changed to "==" operator to make it clear